viewer/packages/camera-manager/src/Flexible/FlexibleCameraManager.ts

```diff
@@ -191,16 +191,10 @@ export class FlexibleCameraManager implements IFlexibleCameraManager {
 
   public update(deltaTime: number, boundingBox: Box3): void {
```

Comment:
Can we rename the second parameter `nearFarBoundingBox`? I feel it makes this a little more clear

Reply:
Fixed